examples/DB/Sqlite3/test_plan.py

```diff
@@ -110,11 +112,14 @@ def main(plan):
     :return: Testplan result object.
     :rtype:  ``testplan.base.TestplanResult``
     """
+    my_db_path = os.path.join(tempfile.gettempdir(), "new_db_name")
     plan.add(
         MultiTest(
             name="Sqlite3Test",
             suites=[DBQueries()],
-            environment=[Sqlite3(name="db", db_name="mydb")],
+            environment=[
+                Sqlite3(name="db", db_name="mydb", db_path=my_db_path)
```

Comment:
It's usage is a bit weird, as ``db_name`` is required, however its value is ignored.

Reply:
We could mark db_name as deprecated and then:

- if db_path is not provided it would default to runpath + driver name
- if db_path is a relative path it would try to connect to runpath + db_path
- if db_path is an absolute path we would try to connect to it

What do you think?

Reply:
Yuxuan suggested a solution, which I think is neat. We could reuse the db_name config option, it can take a relative or an absolute path -- in the former case, we assume it is relative to runpath, in the latter case, we would just use it directly. This would allow user to place the db outside the runpath and persist that.

A more advanced usage would be for user to specify the runpath and path_cleanup=False for the driver.

Reply:
I think this is the same as Marko's suggestion, just reuse the db_name parameter. This way users do not need to change code because of deprecation, but the being the db_name a path it makes confusion for any new user.

Reply:
I agree with Krisztian. Although it would be nice to reuse the db_name argument, it wouldn't really be a "db name" but more like a "path to the db". Other then that your solution is exactly the same as ours so we do agree on that.

I think we should implement your suggestion under an argument called "db_path". Let us know if you disagree.

Reply:
Oh, now I re-read I realize they are indeed the same. Uh... between 'breaking the user' and 'have a confusing param name', i'm slightly leaning towards the latter one. For most of users, they are still going to just provide the base name of the db, and it works like a hidden feature that 'db_name' could take a abs path as well. I think the docstring of this param would really matter to explain the usage and behavior. Anyway, i'm not keen on that, your call.